pages/phases/assessments/reviewing-sme-assessments.md

```diff
@@ -17,7 +17,7 @@ intro: As SMEs submit their interview assessments, check their work and email th
   <div class="usa-alert__body">
     <h3 class="usa-alert__heading">Do not conduct a separate minimum qualification review</h3>
     <p class="usa-alert__text">
-      After SMEs have completed interviews, do not conduct a separate minimum qualification review. For more information, see "<a href="{{ site.baseurl }}/about/differences/">How our process is different</a>."
+      After SMEs have completed assessments, do not conduct a separate minimum qualification review. For more information, see "<a href="{{ site.baseurl }}/about/differences/">How our process is different</a>."
```

Comment:
maybe change "do not" to "you do not need to" ? less prescriptive

Reply:
This is previously existing text. 

I'm ok to change it, but I think it might be better if left - if HR does something separate, it negates what the SME did, right?